utils/crypto.js

```diff
@@ -48,6 +48,26 @@ export const decodePayReq = (payReq, addDefaults = true) => {
   return data
 }
 
+/**
+ * extractMemo - Extracts memo from a payment request.
+ *
+ * @param {string} payReq Payment request
+ * @returns {string|null} Memo
+ */
+export const extractMemo = payReq => {
+  try {
+    const { payment_request } = payReq
```

Comment:
@korhaliv this method might be clearer and more straightforward for a caller to use if it accepted an actual payment request (string) rather than an object that is expected to have a `payment_request` property that contains that string.

Reply:
absolutely